### DISCUSSION TITLE
BROKEN - apt update

### DISCUSSION THREAD

**Discussion by teosayjade** [globaleaks/globaleaks-whistleblowing-software#4362] [General]
apt update command on my linux server is broken after i installed globaleaks.  otherwise, everything is working fine.  please advise.  thanks.

![apt-update](https://github.com/user-attachments/assets/e5f9e547-66a5-4f49-8654-4986334c9d1d)

**Comment by evilaliv3** [comment]
Thank you for notifying this. 

Would you please retry now?

**Comment by teosayjade** [comment]
all is good now.  thanks.  happy holidays!!!
